doc/source/install.rst

```diff
@@ -0,0 +1,25 @@
+Installation
+============
+
+.. attention: The nomenclature requires python >= 3.8
+
+
+Via Pip
+-------
+
+.. code-block:: bash
+
+    pip install nomenclature-iamc
+
+.. attention::  The nomenclature package is distributed as "nomenclature-iamc" on pypi.
```

Comment:
Suggest to move this ahead of the pip-install block...

Reply:
Done